src/llmcompressor/transformers/sparsification/compressed_tensors_utils.py

```diff
@@ -70,55 +141,35 @@ def save_pretrained_wrapper(
             :param kwargs: additional kwargs to pass on to model.save_pretrained
             """
 
+            # patch a shared tensor bug in HF transformers
+            # https://github.com/huggingface/transformers/issues/33689
+            patch_tied_tensors_bug(model)
```

Comment:
This needs to go at the start of oneshot, since we need to untie the tensors before compression starts